### DISCUSSION TITLE
Prism Template Pack visual studio 2022

### DISCUSSION THREAD

**Discussion by developer9969** [PrismLibrary/Prism#2615] [General]
Sorry just realised already been asked

**Comment by dansiegel** [comment]
The move to 64 bit absolutely required extension authors to make updates before the extension will be compatible. Right now both @brianlagunas and I are buried with work from those pesky day jobs. But it'll come as soon as Brian has time to update it.

**Reply by AmenJlili** [reply]
Any updates on this? Thank you all for your hard work!
